pushy-donkeys/Bangazon-Site
# able to delete payment types

Fixes for Ticket #8 
All I did here was add a link in the account settings index to the payment type index.

Steps to test:
```
git checkout master
git fetch --all
git checkout dylan_deletepaymenttype
```
run the app, log in, and click the 'My Account' link.  In there there should be a link at the bottom to manage payment types.  That should take you to the payment type index where you can add and delete or create payment types.

Ticket:

Ticket #8 


@pushy-donkeys
